### PR TITLE
Gami panic vote permissions

### PR DESCRIPTION
**Pull request by bradleyGamiMarques**
* Add internal helper methods.
* Add logic for checking if a user has permissions to use a slash command.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,6 +10,14 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/streemtech/panicbot/internal/slice v1.0.0
```
Comment (deefdragon) [inline go.mod]: This is unnecessary.  It will auto import internal packages as needed.

File: internal/logic/go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module github.com/streemtech/panicbot/internal/logic
```
Comment (deefdragon) [inline internal/logic/go.mod]: Oh. You made them seperate modules.  This is excessive in this case.  You rarely need to use a go mod in another go mod.

File: internal/slice/go.mod [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module github.com/streemtech/panicbot/internal/slice
```
Comment (deefdragon) [inline internal/slice/go.mod]: Same here.  You don't need it to be a seperate module.

File: go.mod (hunk 1 above)
Comment (bradleyGamiMarques) [reply go.mod]: Will I need the replace statements?

**Review by deefdragon — APPROVED** [review]

**Pull request merged**